### ISSUE TITLE
Question: can I use this system with unlabeled data.

### ISSUE THREAD

**Issue by rnunziata** [pyannote/pyannote-audio#167]
Question: can I use this system with unlabeled data.?
I can isolate the speaker segments in an audio stream but not by speaker id.
I need to identify the speakers by id within the stream. 
Can I train system without speaker label data just to identify  the clusters?

**Comment by hbredin** [comment]
For now, `pyannote-audio` cannot use unlabeled data for **training** speech activity detection, speaker change detection, or speaker embedding.

However, there is [a bunch of research work](https://github.com/pyannote/pyannote-audio/issues/50) trying to do this (in particular for speaker embedding).

That being said, you [can](https://github.com/pyannote/pyannote-audio/tree/develop/tutorials/models/pretrained) use pre-trained models (either provided in this repo, or trained by yourself on a similar labeled dataset) and apply them on unlabeled data.

**Comment by rnunziata** [comment]
How do use the pre-trained models to test this ... can I just create a annotation for the wav file and add it to the dict  as you suggest in issues/166?

**Comment by hbredin** [comment]
Just follow the tutorial and get rid of everything related to visualizing the "annotation" key.

**Comment by hbredin** [comment]
Closing. Please re-open if needed.
